.github/workflows/charts-ci.yaml

```diff
@@ -36,9 +33,15 @@ jobs:
         with:
           version: v1.18.2-k3s1
 
-      - name: Run chart-testing (install)
-        uses: helm/chart-testing-action@v1.0.0
+  integration-tests:
+    name: integration-tests
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v1
         with:
-          command: install
-          config: ct-master.yaml
-          kubeconfig: ${{ steps.k3s.outputs.kubeconfig }}
+          fetch-depth: 1
+     
+      - name: Run Pulsar Admin Console tests
+        run: |
+          ./charts/tests/run-tests.sh pulsar-admin-console
```

Comment:
Would it be easy to make this only run when changes are made to the pulsar admin console chart?

Reply:
Fixed